app/assets/stylesheets/common/base/header.scss

```diff
@@ -15,6 +15,23 @@
       display: flex;
       align-items: center;
       height: 100%;
+
+      .header-row {
+        width: 100%;
+
+        .logo-wrapper {
```

Comment:
One thing to consider here is using flexbox. It's not required for this PR but I find it's much easier to wrap your head around than floats.

Reply:
Noted. Will start using that.